wagtail/wagtailadmin/menu.py

```diff
@@ -31,10 +33,22 @@ def is_shown(self, request):
         """
         return True
 
+    def is_active(self, request):
+        return request.path.startswith(self.url)
+
     def render_html(self, request):
-        return format_html(
-            """<li class="menu-item menu-{0}"><a href="{1}" class="{2}"{3}>{4}</a></li>""",
-            self.name, self.url, self.classnames, self.attr_string, self.label)
+        t = loader.get_template(self.template)
+        c = Context({
+            'name': self.name,
+            'url': self.url,
+            'classnames': self.classnames,
+            'attr_string': self.attr_string,
+            'label': self.label,
+            'request': request,
+            'active': self.is_active(request)
+        })
+
+        return t.render(c)
```

Comment:
Using Template.render with a Context object is deprecated in Django 1.8. https://docs.djangoproject.com/en/1.8/ref/templates/upgrading/#get-template-upgrade-django-18

Could I recommend using the ``render_to_string`` function defined in ``wagtail.utils.compat``?

```python
from wagtail.utils.compat import render_to_string

def render_html(self, request):
    return render_to_string(self.template, {
        'name': self.name,
        'url': self.url,
        'classnames': self.classnames,
        'attr_string': self.attr_string,
        'label': self.label,
        'active': self.is_active(request)
    }, request=request)
```